libs/design/src/molecules/hero/hero.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, ViewEncapsulation, Input } from '@angular/core';
+
+@Component({
+  selector: 'daff-hero',
+  templateUrl: './hero.component.html',
+  styleUrls: ['./hero.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Is there a way to avoid `ViewEncapsulation.None` here?

Reply:
I think that it highly like that we use nested components inside heroes. I'd argue that we can move the hero title and the hero subtitle styles into their component styles, that way we can keep view encapsulation set to emulated.

Reply:
After thinking about this, I think we can get away with `ViewEncapsulation.None` due to the BEM syntax that we're fairly strict about.